packages/modelserver-node/src/test/model-service-integration-test.spec.ts

```diff
@@ -10,43 +10,40 @@
  *******************************************************************************/
 import {
     CompoundCommand,
-    Diagnostic,
-    ModelServerCommand,
     ModelServerNotificationListenerV2,
     ModelServerObjectV2,
     NotificationSubscriptionListenerV2,
-    Operations,
-    SetCommand,
-    SubscriptionListener,
     WARNING
 } from '@eclipse-emfcloud/modelserver-client';
-import {
-    CommandProvider,
-    ModelServerClientApi,
-    ModelService,
-    ModelServiceFactory,
-    TriggerProvider
-} from '@eclipse-emfcloud/modelserver-plugin-ext';
+import { ModelServerClientApi, ModelService, ModelServiceFactory } from '@eclipse-emfcloud/modelserver-plugin-ext';
 import * as chai from 'chai';
 import { expect } from 'chai';
 import * as chaiLike from 'chai-like';
 import { getValueByPointer, Operation } from 'fast-json-patch';
 import { Container } from 'inversify';
-import { Context } from 'mocha';
 import * as URI from 'urijs';
 
 import { CommandProviderRegistry } from '../command-provider-registry';
-import { awaitClosed, CoffeeMachine, isCoffeeMachine, ServerFixture } from '../server-integration-test.spec';
 import { TriggerProviderRegistry } from '../trigger-provider-registry';
 import { ValidationProviderRegistry } from '../validation-provider-registry';
+import {
+    _assumeThat,
+    awaitClosed,
+    findDiagnostic,
+    listenForFullUpdate,
+    registerCommand,
+    registerTrigger,
```

Comment:
I'm not sure that these are useful in any other context. Did they need to be extracted to a separate file?

Reply:
Agreed, moved them back again.